extensions/azure-functions-http/runtime/src/main/java/io/quarkus/azure/functions/resteasy/runtime/BaseFunction.java

```diff
@@ -156,6 +158,11 @@ public void handleMessage(Object msg) {
                 }
                 if (msg instanceof LastHttpContent) {
                     responseBuilder.body(baos.toByteArray());
+                    try {
+                        baos.close();
```

Comment:
Look at the implementation for BAOS.close().  its empty.

Reply:
Oh yes !
So I'd rather remove the closing of the BAOS from google-cloud-functions-http